packages/server/src/routes/auth/handlers/registerUser.ts

```
@@ -0,0 +1,30 @@
+import { DBConnection } from "../../../drizzle/drizzle-client";
+import { authTable, type SelectAuth } from "../../../drizzle/schema";
+import { eq } from "drizzle-orm";
+
+export class AuthTable extends DBConnection {
+	constructor(TURSO_CONNECTION_URL: string, TURSO_AUTH_TOKEN: string) {
+		super(TURSO_CONNECTION_URL, TURSO_AUTH_TOKEN);
+	}
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)